### DISCUSSION TITLE
Can I connect an address and sign a transaction from nodejs without a browser in Wagmi core

### DISCUSSION THREAD

**Discussion by ImCitizen13** [wevm/wagmi#1745] [General]
Hi there, 

I'm building a nestjs backend and I want to execute a **write** transaction to the contract as the contract owner, 
I couldn't find anyway to connect my address and sign the transaction programmatically to execute a write task without 
needing a browser.

**Comment by Raiden1411** [comment]
Use `ethers.wallet` for this. You will need a private key. Check their docs for more info
